src/inc/DefaultSettings.h

```diff
@@ -14,14 +14,12 @@ Author(s):
 --*/
 #pragma once
 
-constexpr COLORREF COLOR_WHITE = 0x00ffffff;
-constexpr COLORREF COLOR_BLACK = 0x00000000;
+constexpr til::color COLOR_WHITE{ 0xff, 0xff, 0xff };
+constexpr til::color COLOR_BLACK{ 0, 0, 0 };
 constexpr COLORREF OPACITY_OPAQUE = 0xff000000;

```

Comment:
OPQCITY_OPAQUE is still used in the dx engine, which has not onboarded to til::color.